systers_portal/templates/meetup/snippets/user-cell.html

```diff
@@ -0,0 +1,15 @@
+{% load staticfiles %}
+
+{% if user.profile_picture.name and user.profile_picture.name != "False" %}
+  <a href="{{ user.get_absolute_url }}">
+    <img src="{{ user.profile_picture.url }}" class="profile-pic img-square img-responsive img-wh-100" alt="{{ user }} profile picture"/>
+  </a>
+{% else %}
+  <a href="{{ user.get_absolute_url }}">      
+    <img src="{% static 'img/default.png' %}" class="profile-pic img-square img-responsive img-wh-100"
```

Comment:
So the images look tiny because you apply the class `profile-pic`, it sets the width to 50%. Remove it. Should have looked tiny both in Chrome and in Firefox (on Chrome the cache was the problem probably). Remove `img-square` and `img-wh-100`. Once you remove them you will still need to set some margin between the pic and user name.